fenix/app/src/main/res/values/strings.xml

```diff
@@ -65,6 +65,17 @@
        Common myths about private browsing
     </string>
 
+    <!-- True Private Browsing Mode -->
+    <!-- Title for description card on private homescreen in True Private Browsing Mode. -->
+    <string name="felt_privacy_desc_card_title">Leave no traces on this device.</string>
+    <!-- Explanation for private browsing displayed to users on home view when they first enable
+        private mode in our new Total Private Browsing mode.
+        The first parameter is the name of the app defined in app_name (for example: Fenix) -->
+    <string name="felt_privacy_desc_card_subtitle">%1$s deletes your cookies, history, and site data when you close all your private windows. Who might be able to see my activity?</string>
```

Comment:
See the conversation happening [here](https://github.com/mozilla-mobile/firefox-android/pull/3390#issuecomment-1702429103). Any chance we could keep this as a single string and add markup?

Reply:
@flodolo I am wondering if this is strictly required? Is the concern around having 2 many placeholders?

For reference, this is the string that we want to display:
<img width="608" alt="Screen Shot 2023-09-05 at 2 37 07 PM" src="https://github.com/mozilla-mobile/firefox-android/assets/1190888/0cd1838f-7272-40c1-9be5-ccc2a97280af">

Reply:
In this case the second string is a complete string, so there is little risk. 

In the PR I linked to, the link is a piece of sentence or word that ends up being translated in a separate string, and localizers don't translate them together. The risk of creating a full sentence that doesn't make a lot of sense, or just has punctuation/capitalization problems, is not zero.

Reply:
Since the second string is a complete string and there is little risk, would you still advise with adding markup or could we proceed with adding details that reference the parameter and their usage to help translator build the context for a full translation of the strings together?

Reply:
In general, having a single string exposed with markup would always be better. If that requires too much time to investigate, then I wouldn't block this specific PR.

Reply:
With the current implementation of `LinkText`, it's necessary to have a second string of just the clickable portion of the first string so it can be properly formatted and clickability assigned.
Both the non-clickable and clickable portions of this particular text are separate sentences so it shouldn't be too hard to translate.